### DISCUSSION TITLE
Mobile instruction manual

### DISCUSSION THREAD

**Discussion by reckoner-exe** [community/community#147271] [Mobile]
### Select Topic Area

Show & Tell

### Body

hi I have been told this is the instruction manual for the github. 

https://drive.google.com/file/d/1tOpiReFOGf9_Ic5d7DiC0LDZCM23FSVT/view?usp=sharing

**Comment by eliperkins** [comment]
Hi @reckoner-exe , we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members."
